ocp/README.md

```diff
@@ -0,0 +1,19 @@
+## IBM Z and Cloud Modernization Stack - zmodstack-deploy
+This repository contains automation scripts for deploying IBM Z and Cloud Modernization Stack components in various environments.
+
+The different roles specify tasks to deploy the individual components.
+To execute the playbooks, adhere to the instructions below:
+
+## Step 1: Ensure that the prerequisites are prepared.
+** 1 ** Copy the kubeconfig file to the /root/.kube/ directory with the filename "config." This choice is due to the fact that /root/.kube/config is where the ansible module will naturally search for the kubeconfig file.
+
+** 2 ** clone the OCP repository using the following commands that will do sparse checkout.
+# Sparse checkout to clone ocp mono repo
+git clone --no-checkout https://github.com/IBM/zmodstack-deploy.git /home/ec2-user/zmodstack/
+cd /home/ec2-user/zmodstack && git sparse-checkout set ocp && git checkout @
```

Comment:
Since the branch was not specified the default branch was being clone that did not contain the "ocp" mono repo. 
I have added this correction in the following PR